FreeRTOS/FreeRTOS
# WIN32-MSVC Demo Re-work interrupt handling.

Description
-----------


This change uses the simulated interrupts from WinSim port instead of using a polling task to check for keyboard presses.

Test Steps
-----------


Built both main and blinky and visual studio 2019.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
